types/node/node-tests.ts

```diff
@@ -1403,6 +1403,17 @@ namespace dgram_tests {
             let _rinfo: dgram.AddressInfo = rinfo;
         });
     }
+
+    {
+        var ds: dgram.Socket = dgram.createSocket({
+            type: 'udp4',
+            recvBufferSize: 10000,
+            sendBufferSize: 15000
+        });
+
+        console.log(ds.getRecvBufferSize());
```

Comment:
`console.log()` is actually not testing anything here. I recommend to use something like `let size: number = ds.getRecvBufferSize()` as this verifies that we get a number.